Sources/PrimerSDK/Classes/Error Handler/PrimerError.swift

```diff
@@ -457,6 +58,7 @@ public enum PrimerError: PrimerErrorProtocol {
     case sdkDismissed
     case failedToProcessPayment(paymentId: String, status: String, userInfo: [String: String]?, diagnosticsId: String)
     case invalidVaultedPaymentMethodId(vaultedPaymentMethodId: String, userInfo: [String: String]?, diagnosticsId: String)
+    case nolError(code: String?, message: String?, userInfo: [String: String]?, diagnosticsId: String)
```

Comment:
Is this the right pattern? I don't see specific error types for Klarna, iPay88, etc.

Reply:
Can you suggest any name ideas?

wdyt @semirp?

Reply:
Leave it as is for now - we should review all of the errors (similarly to tracking events) in the near future. Then we can decide if we want to break it down or introduce per-payment-method errors.